index.html

```diff
@@ -0,0 +1,160 @@
+    <div class="program-part">
+      <div class="program-items">
+        <img src="assets/program_icon_01 1.png" alt="Neural Networking Image" />
+        <h4>Neural Networking</h4>
+        <p>
+          Graph Neural Networking challenge 2023: Building a Network Digital Twin using data from real networks.
+        </p>
+      </div>
+      <div class="program-items">
+        <img src="assets/program_icon_02 1.png" alt="Machine Learning Image" />
+        <h4>Machine Learning</h4>
+        <p>
+          Incorporate machine learning tools into a network can help teams predict traffic flows, generate smarter
+          analytics, monitor network health, tighten security measures and more. </p>
+      </div>
+      <div class="program-items">
+        <img src="assets/program_icon_03 1.png" alt="Augmented Reality image" />
+        <h4>Augumented Reality</h4>
+        <p>
+          Learn AI algorithms can be used in real-time to detect and track objects, allowing AR and VR experiences to
+          adapt to the user's movements and actions
+        </p>
+      </div>
+      <div class="program-items">
+        <img src="assets/program_icon_04 1.png" alt="Promt engineering image" />
+        <h4>Prompt Engineering</h4>
+        <p>
+          Discover top prompt engineering jobs on our niche job board, tailored to connect innovative companies with
+          skilled professionals in the field.
+        </p>
+      </div>
+      <div class="program-items">
+        <img src="assets/program_icon_05 1.png" alt="generative AI image" />
+        <h4>Generative AI</h4>
+        <p>
+          Generative AI is a type of artificial intelligence technology that can produce various types of content,
+          including text, imagery, audio and synthetic data.
+        </p>
+      </div>
+
+    </div>
+
+    <button class="program-btn" type="button"><strong>Join AIFG India Summit 2023</strong></button>
+    <a href="#" class="click-to-see"><strong>SEE THE WHOLE PROGRAM</strong></a>
+
+  </section>
+  <!-- program section end -->
+  <!-- speakers start -->
+  <section class="program-speakers" id="program-speakers">
+    <h3 class="program-speakers-title"><strong>Featured Speakers</strong></h3>
+
+    <div class="program-speaker-part"></div>
+    <button type="button" class="program-speaker-btn">More <img src="assets/arrow_down.png" alt="arrow_down"></button>
+  </section>
+  <!-- speakers end -->
+  <section class="our-partner" id="partner"> 
+    <h3 class="our-partner-title">Partner</h3>
+     <div class="our-partner-underline" ></div>
+    <div class="our-partner-part">
+       <div class="our-partner-logo">
+       <img src="assets/py.png" alt="partner 1">
+       <img src="assets/unesco.png" alt="partner 2">
+       <img src="assets/bill.png" alt="partner 3">
+       <img src="assets/unicef.png" alt="partner 4">
+       <img src="assets/un.png" alt="partner 5">
+       <img src="assets/imf.png" alt="partner 6">  
+     </div>
```

Comment:
On mobile, your partner images should be properly aligned to match the given design as it is a requirement in this milestone that we stick to the given design😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="323" alt="image" src="https://github.com/the-faizmohammad/conferencePage/assets/64643995/2d8fd3b7-4363-4729-9f10-ca5d5766c12e">|<img width="319" alt="image" src="https://github.com/the-faizmohammad/conferencePage/assets/64643995/12666484-1afb-43c4-a19a-c1a28ac85ecb">|